playground/src/editor/editor.js

```diff
@@ -68,6 +70,14 @@ class Editor {
     this.errorMarkers = [];
     this.loader = new Loader(this.container, 'light');
     this.amphtmlHints = this.fetchHintsData();
+
+    events.subscribe(ImportURL.EVENT_UPDATE_EDITOR_CONTENT, (html) => {
+      window.requestIdleCallback(() => {
+        if (html) {
```

Comment:
Ideally the event isn't even fired when `html == undefined`. Don't you handle the error case somewhere else anyway?